framework/wazuh/core/batcher/batcher.py

```diff
@@ -0,0 +1,196 @@
+import asyncio
+import uuid
+from typing import List, Optional
+from multiprocessing import Process
+import queue
+import signal
+import traceback
+import logging
```

Comment:
Imports order should conform with the PEP8 style guide

```suggestion
import asyncio
import uuid
import queue
import signal
import traceback
import logging
from typing import List, Optional
from multiprocessing import Process
```

Reply:
Fixed in https://github.com/wazuh/wazuh/commit/47e4508357e0fb53f499847d6a791a0f4c86f15b.